libs/remix-ws-templates/src/templates/remixDefault/README.txt

```diff
@@ -2,25 +2,27 @@ REMIX DEFAULT WORKSPACE
 
 Remix default workspace is present when:
 i. Remix loads for the very first time 
-ii. A new workspace is created
+ii. A new workspace is created with 'Default' template
 iii. There are no files existing in the File Explorer
 
 This workspace contains 3 directories:
 
 1. 'contracts': Holds three contracts with different complexity level, denoted with number prefix in file name.
-2. 'scripts': Holds two scripts to deploy a contract. It is explained below.
-3. 'tests': Contains one Solidity test file for 'Ballot' contract & one JS test file for 'Storage' contract
+2. 'scripts': Contains four typescript files to deploy a contract. It is explained below.
+3. 'tests': Contains one Solidity test file for 'Ballot' contract & one JS test file for 'Storage' contract.
 
 SCRIPTS
 
-The 'scripts' folder contains two example async/await scripts for deploying the 'Storage' contract.
-For the deployment of any other contract, 'contractName' and 'constructorArgs' should be updated (along with other code if required). 
+The 'scripts' folder have four typescript files which help to deploy the 'Storage' contract using 'web3.js' and 'ethers.js' libraries.
 
-Also, there is a script containing some unit tests for Storage contract inside tests directory.
+For the deployment of any other contract, just update contract name from 'Storage' to desired contract and provide constructor arguments accordingly 
+in the file `deploy_with_ethers.ts` or  `deploy_with_web3.ts`
+
+Also, there is a script containing some Mocha-Chai unit tests for 'Storage' contract inside 'tests' directory.
 
 To run a script, right click on file name in the file explorer and click 'Run'. Remember, Solidity file must already be compiled.
 Output from script will appear in remix terminal.
 
-Please note, 'require' statement is supported in a limited manner for Remix supported modules.
-For now, modules supported by Remix are ethers, web3, swarmgw, chai, remix and hardhat only for hardhat.ethers object/plugin.
-For unsupported modules, an error like this will be thrown: '<module_name> module require is not supported by Remix IDE will be shown.'
\ No newline at end of file
+Please note, require/import is supported in a limited manner for Remix supported modules.
+For now, modules supported by Remix are ethers, web3, swarmgw, chai, multihashes, remix and hardhat only for hardhat.ethers object/plugin.
```

Comment:
I'm not sure that swarm still supports swarmgw - but lets leave it for now.

Reply:
ok